extensions/amp-form/0.1/amp-form.js

```diff
@@ -1222,7 +1226,9 @@ export class AmpForm {
    * @private
    */
   analyticsEvent_(eventType, opt_vars) {
-    triggerAnalyticsEvent(this.form_, eventType, opt_vars);
+    // Use cached `this.ampdoc_` to trigger analytics since analytics
+    // may be called after the <form> has been removed from the DOM.
+    triggerAnalyticsEvent(this.ampdoc_, eventType, opt_vars);
```

Comment:
Is `ampdoc_` an element?

Reply:
Sure!  I'll revert that change since the issue with `<form>` is fixed w/ just the try/catch block.
That said, there are still bugs here that I think should be addressed.

What should happen to analytics calls where the element is no longer on the DOM? Should we drop them? Should we pass in a special option specifying the ampdoc so that it doesn't go searching?

Reply:
That's a good question. Just for more clarification, the form is disappearing from the DOM? Why is that?

Reply:
amp-form can appear within an amp-list's template. so when the amp-list rerenders, the amp-form may disappear

Reply:
Ahh, tracking dynamically added elements has always been on a case-by-case basis for `amp-analytics`. The general solution is to include a new analytics element within `amp-list`. In this example where the error occurs, is that the case? Or is the analytics element outside of `amp-list`?

Reply:
analytics element is outside. does that mean the page would have two analytics elements? the issue here isn't the dynamic add, but is actually the removal.

Reply:
Yup, multiple analytics elements on a page is fine.